### DISCUSSION TITLE
Center protein after conversion.

### DISCUSSION THREAD

**Discussion by pachequin1** [owenvickery/cg2at#24] [Q&A]
Hi, I was trying cg2at with a protein/membrane system and everything runs fine (thank you) but I want to know if there is any flag to move the final protein structure at the center box.

Is it possible?

Thank you

Cesar

**Comment by owenvickery** [comment]
Hi,

Good to hear!

I didn't put that in because the box compositions are so varied that it invariable goes wrong. Easiest way would be to use editconf on the output. I might do it when I do another rewrite of the script.

Cheers
Owen
